sql/2024/privacy/ads_and_sellers_graph.sql

```diff
@@ -0,0 +1,114 @@
+WITH RECURSIVE pages AS (
+  SELECT
+    CASE page -- publisher websites may redirect to an SSP domain, and need to use redirected domain instead of page domain
+      WHEN 'https://www.chunkbase.com/' THEN 'cafemedia.com'
+      ELSE NET.REG_DOMAIN(page)
+    END AS page,
+    custom_metrics
+  FROM `httparchive.all.pages`
+  WHERE date = '2024-06-01' AND
+    client = 'mobile' AND
```

Comment:
Why only `mobile`?

I don't _think_ they should be any difference between mobile and desktop, but that's an assumption we're making here so any harm to run both and confirm that assumption?

Reply:
The query was failing by hitting time limits.
But seems I've made optimisation in another place and we could include both indeed.

Anyway, the team decided to skip ads transparency analysis in the chapter.

Reply:
Oh I noticed a few places where this was used and assumed the whole chapter was mobile-only. But seems I judged too much and it's only 4 queries that are restricted o mobile. So not a big deal.